editors/vscode/src/extension.ts

```diff
@@ -152,6 +152,8 @@ function startClient(
     );
     options.env["SLINT_LSP_PANIC_LOG"] = slint_lsp_panic_file.fsPath;
 
+    options.cwd = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath;
```

Comment:
This is already the default, but it can't hurt: 
https://github.com/microsoft/vscode-languageserver-node/blob/4376df05e1cc7cefb106137f74b844fc709c8e7a/client/src/node/main.ts#L576